ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ViolationsByPolicyCategory.tsx

```diff
@@ -142,36 +168,112 @@ function ViolationsByPolicyCategoryChart({ alertGroups }: ViolationsByPolicyCate
+
+const fieldIdPrefix = 'policy-category-violations';
+
 function ViolationsByPolicyCategory() {
+    const { isOpen: isOptionsOpen, onToggle: toggleOptionsOpen } = useSelectToggle();
     const { searchFilter } = useURLSearch();
-    const query = getRequestQueryStringForSearchFilter(searchFilter);
+    const [sortType, sortTypeOption] = useState<SortTypeOption>('Severity');
+    const [lifecycle, setLifecycle] = useState<LifecycleOption>('All');
+
+    const queryFilter = { ...searchFilter };
+    if (lifecycle === 'Deploy') {
+        queryFilter['Lifecycle Stage'] = LIFECYCLE_STAGES.DEPLOY;
+    } else if (lifecycle === 'Runtime') {
+        queryFilter['Lifecycle Stage'] = LIFECYCLE_STAGES.RUNTIME;
+    }
+    const query = getRequestQueryStringForSearchFilter(queryFilter);
     const { alertGroups, loading, error } = useAlertGroups('CATEGORY', query);
 
     return (
         <WidgetCard
             isLoading={loading}
             error={error}
             header={
-                <Title headingLevel="h2" className="pf-u-p-md">
-                    Policy violations by category
-                </Title>
+                <Flex direction={{ default: 'row' }} className="pf-u-pb-md">
+                    <FlexItem grow={{ default: 'grow' }}>
+                        <Title headingLevel="h2">Policy violations by category</Title>
+                    </FlexItem>
+                    <FlexItem>
+                        <Dropdown
+                            toggle={
+                                <DropdownToggle
+                                    id={`${fieldIdPrefix}-options-toggle`}
+                                    toggleVariant="secondary"
+                                    onToggle={toggleOptionsOpen}
+                                >
+                                    Options
+                                </DropdownToggle>
+                            }
+                            position="right"
+                            isOpen={isOptionsOpen}
+                        >
+                            <Form className="pf-u-px-md pf-u-py-sm">
+                                <FormGroup fieldId={`${fieldIdPrefix}-sort-by`} label="Sort by">
+                                    <ToggleGroup aria-label="Sort data by highest severity counts or highest total violations">
+                                        <ToggleGroupItem
+                                            className="pf-u-font-weight-normal"
+                                            text="Severity"
+                                            buttonId={`${fieldIdPrefix}-sort-by-severity`}
+                                            isSelected={sortType === 'Severity'}
+                                            onChange={() => sortTypeOption('Severity')}
+                                        />
+                                        <ToggleGroupItem
+                                            text="Volume"
```

Comment:
Volume takes a second, and then a third thought, to understand: does it mean sort by total violations?

Reply:
There was a comment in the design doc that the terminology for "Volume" might be confusing, I'll bring this up in the channel as well.